go/internal/openapi/client.go

```diff
@@ -64,12 +64,12 @@ type APIClient struct {
 
 	EnvironmentApi *EnvironmentApiService
 
-	EnvironmentSettingsApi *EnvironmentSettingsApiService
```

Comment:
I wonder if this was intentional, either to have it be in the lib at all in the first place, or the removal now.

Does anyone know the story here?

Reply:
Hmm, no idea. From looking at the code, it seems the same methods are available in EnvironmentApi. It doesn't look like we are loosing any functionality

Reply:
Sure. It's still a breaking change I think. I just want to be sure it's intended breakage, but I don't have the context around this change.

Reply:
No longer happening after Aaron's change